lib/postgrex.ex

```diff
@@ -66,6 +66,13 @@ defmodule Postgrex do
     if not `DBConnection.Connection` (default: `DBConnection.Connection`);
     * `:null` - The atom to use as a stand in for postgres' `NULL` in encoding
     and decoding (default: `nil`);
+    * `:date` - Either `:postgrex` to use Postgrex date structs or
```

Comment:
@ericmj are we sure we don't want to require Elixir v1.3 by default and default to `:elixir` and keep the `:postgrex` ones only for backwards compatibility?